ngi_pipeline/utils/test_filesystem.py

```diff
@@ -42,6 +43,19 @@ def test_do_rsync(self):
         for dst_file_path in dst_file_paths:
             assert(os.path.exists(dst_file_path))
 
+    def test_do_links(self):
+        src_tmp_dir = tempfile.mkdtemp()
+        dst_tmp_dir = os.path.join(src_tmp_dir, 'dst' )
+        safe_makedir(dst_tmp_dir)
+        src_file_path = os.path.join(src_tmp_dir, 'file1.txt') 
+        dst_file_path = os.path.join(dst_tmp_dir, 'file1.txt') 
+        open(src_file_path, 'w').close()
+        do_hardlink([src_file_path], dst_tmp_dir)
+        assert(filecmp.cmp(src_file_path, dst_file_path))
+        os.remove(dst_file_path)
+        do_symlink([src_file_path], dst_tmp_dir)
+        assert(filecmp.cmp(src_file_path, dst_file_path))
```

Comment:
That would not prove that the two files are the same, it could be a link to anything, for instance, a broken link to a forgotten file.